src/controllers/todo-controller.ts

```diff
@@ -29,6 +29,17 @@ export class TodoController extends BaseController {
     );
 
     this.router.delete(`${this.basePath}/:id`, this.deleteTodo);
+<<<<<<< Updated upstream
```

Comment:
Seems like merge conflicts are not resolved.

Reply:
yes, fixing that.